src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
+            "sequenceReference": {
+              "id": "NM_004327.4",
+              "type": "SequenceReference",
+              "refgetAccession": "SQ.kpytJsXw3BwLC3oBSjHQS1kwxs4WO3I3",
+              "residueAlphabet": "na"
             },
-            "end": {
-              "type": "Number",
-              "value": 130854065
-            }
+            "end": 3234
+          }
+        },
+        {
+          "name": "gene",
+          "description": "The gene concept (BCR) associated with this fusion partner.",
+          "value": {
+            "code": "hgnc:1014",
+            "system": "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/",
+            "label": "BCR"
           }
         }
-      }
+      ]},
+      {
+        "type": "SequenceLocation",
+        "sequenceReference": {
+          "id": "GRCh38:chr9",
+          "type": "SequenceReference",
+          "refgetAccession": "SQ.KEO-4XBcm1cxeo_DIQ8_ofqGUkp4iZhI",
+          "residueAlphabet": "na"
+        },
+        "start": 130854064,
+        "extensions": [
+          {
+            "name": "NM_005157.6:e.2_",
+            "description": "VICC exon representation of the aligned transcript boundary.",
+            "value": {
+              "exon_start": 2,
+              "exon_start_offset": 0,
+              "sequenceReference":{
+                "id": "NM_005157.6",
+                "type": "SequenceReference",
+                "refgetAccession": "SQ.w8Qg3x-PQ2akJrJQeGEN-_eBUMo1H1CL",
+                "residueAlphabet": "na"
+              }
+            }
+          },
+          {
+            "name": "NM_005157.6:c.80_",
+            "description": "Transcript SequenceLocation of the aligned transcript boundary.",
+            "value": {
+              "type": "SequenceLocation",
+              "sequenceReference": {
+                "id": "NM_005157.6",
+                "type": "SequenceReference",
+                "refgetAccession": "SQ.w8Qg3x-PQ2akJrJQeGEN-_eBUMo1H1CL",
+                "residueAlphabet": "na"
+              },
+              "end": 273
+            }
+          },
+          {
+            "name": "gene",
+            "description": "The gene concept (ABL1) associated with this fusion partner.",
+            "value": {
+              "code": "hgnc:76",
+              "system": "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/",
+              "label": "ABL1"
+            }
+          }
+        ]
+      }],
+    "linker": {
+      "type": "LiteralSequenceExpression",
+      "sequence": "CCCGTC"
     }
-  ],
-  "r_frame_preserved": true,
-  "critical_functional_domains": [
+  },
+  "readingFramePreserved": true,
+  "criticalFunctionalDomains": [
     {
       "type": "FunctionalDomain",
       "status": "preserved",
-      "associated_gene": {
-        "id": "normalize.gene:hgnc%3A76",
-        "type": "GeneDescriptor",
-        "label": "ABL1",
-        "gene_id": "hgnc:76"
+      "gene": {
```

Comment:
mostly noting this to help myself as I am working on the updates to fusor - this should be `associatedGene`